docs/products/clickhouse/reference/limitations.rst

```diff
@@ -52,6 +52,9 @@ From the information about restrictions on using Aiven for ClickHouse, you can e
    * - Creating a database using SQL
      - You cannot create a database directly using SQL, for example, if you'd like to add a non-default database.
      - Use the Aiven's public API.
+   * - Scaling down a sharded cluster
+     - You only can scale up between shards. Scaling down is not supported with the sharding strategy. You cannot downgrade to a service plan with fewer VMs if your ClickHouse cluster is sharded.
```

Comment:
You can "scale up" or "scale down" what, exactly?
I just checked, and it _is_possible to scale down storage, cpu and RAM, in a sharded cluster, if disk space allows that.

The only thing that is unsupported is scaling down the number of nodes. I don't know if this is supported for unshared clusters by the way, because I wasn't able to find plans with more than 3 nodes and just 1 shard, so the point is kind of "undecidable" right now.